opensrp-app/src/main/java/org/smartregister/sync/intent/SyncIntentService.java

```diff
@@ -213,6 +236,8 @@ private void processFetchedEvents(Response resp, ECSyncHelper ecSyncUpdater, fin
             }
             sendSyncProgressBroadcast(eCount);
             fetchRetry(0, false);
+            addAttribute(eventSyncTrace, COUNT, String.valueOf(eCount));
+            stopTrace(eventSyncTrace);
```

Comment:
Sync trace should be stopped here `boolean isSaved = ecSyncUpdater.saveAllClientsAndEvents(jsonObject);`

We  should add another trace on this line ` processClient(serverVersionPair);` This will allow to know how much client processing takes